README.md

```diff
@@ -470,6 +470,7 @@ If the subscriber has already been aborted (i.e., `subscriber.signal.aborted` is
  - From `complete()`, after the subscriber's complete handler (if any) is
    invoked
  - From `error()`, after the subscriber's error handler (if any) is invoked
+ - The signal passed to the subscription is aborted by the user.
```

Comment:
Technically I don't think this was needed because of https://github.com/domfarolino/observable/pull/61/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R466-R467 (i.e., we're already in the "Otherwise" branch here) but I guess it can't hurt?